### PR TITLE
Behat integration

### PR DESCRIPTION
**Pull request by Opio-Emmanuel-Omona**

### CODE REVIEW

File: features/bootstrap/RatingContext.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    $this->artisan('migrate');
+  }
+  /** @AfterScenario */
+  public function after(AfterScenarioScope $scope)
+  {
+    $this->artisan('migrate:rollback');
+  }
+
+  /**
+   * @Given I am a user with permission: :arg1
```
Comment (naush) [inline features/bootstrap/RatingContext.php]: Is it possible to rename the variable? `:permission` could be good for this argument, and `:path` or `:url` for the other function.

File: features/rating-auth.feature [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Feature: Rating Authentication
+  Scenario: Cannot rate a church without permission
+    Given I am a user with permission: ''
```
Comment (naush) [inline features/rating-auth.feature]: I wonder if this line would read better as: `I am a user without any permission`  or `I am a user with permission: 'not_rate_questions'`.

File: features/bootstrap/FeatureContext.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?php
```
Comment (naush) [inline features/bootstrap/FeatureContext.php]: Is this file required even if we don't use it?

File: features/bootstrap/RatingContext.php (hunk 1 above)
Comment (Opio-Emmanuel-Omona) [reply features/bootstrap/RatingContext.php]: Yap, it should be, let me change that

**Review by naush — APPROVED** [review]
LGTM! 🚢
